src/sentry/static/sentry/app/views/organizationDashboard/discoverQuery.jsx

```diff
@@ -9,16 +9,27 @@ import {parsePeriodToHours} from 'app/utils';
 import SentryTypes from 'app/sentryTypes';
 import createQueryBuilder from 'app/views/organizationDiscover/queryBuilder';
 
+// Note: Limit max releases so that chart is still a bit readable
+const MAX_RECENT_RELEASES = 20;
```

Comment:
👍

Reply:
Just curious, is there a specific reason why we are increasing from 10 to 20?

Reply:
Was testing this for staging, will return it back to 10, but the idea is that if you release often and you're looking at a sufficiently long time period, your chart ends up looking like: 
![image](https://user-images.githubusercontent.com/79684/56537018-561cf380-6514-11e9-9426-69d97f999d72.png)